src/Synapse/Synapse/Common/JsonConvert.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.IO;
+using System.Reflection;
+using System.Text.Json;
+
+namespace Microsoft.Azure.Commands.Synapse.Common
+{
+    internal static class JsonConvert
+    {
+        internal static T DeserializeObject<T>(string rawJsonContent)
+        {
+            var document = JsonDocument.Parse(rawJsonContent, default);
```

Comment:
Feature `default literal` is not available in C# 7.0.

Reply:
Thanks for the suggestion. Fixed.